lib/dispatch_web/webhooks/controller.ex

```diff
@@ -25,7 +25,8 @@ defmodule DispatchWeb.Webhooks.Controller do
     repo = get_in(params, ["repository", "full_name"])
 
     stacks = Dispatch.extract_from_params(params)
-    selected_users = Dispatch.fetch_selected_users(repo, stacks, author)
+    mute_learners = Map.get(params, "mute_learners")
```

Comment:
We should use `string_to_boolean` right here, not when we pass the fourth argument to `Dispatch.fetch_selected_users/4`.